.github/workflows/push-release.yml

```
@@ -52,27 +52,36 @@ jobs:
           overwrite: true
   publish-chart:
     runs-on: ubuntu-latest
+    needs: build-chart
```

```suggestion
    needs: [build-chart, push-images]
```
to avoid publishing the chart before publishing the docker images